### DISCUSSION TITLE
systemctl failure

### DISCUSSION THREAD

**Discussion by matter1-git** [termux/termux-packages#17585] [Q&A] [answered]
Hello guys,

I have recently downloaded and played around with termux and found it amazing.

I do have a problem when installing snapd. The service does not start post installation and the systemctl breaks.

Here is the error message:

system has not been booted with systemd as init system (pid 1). can't operate

Q: Is this a bug worth submitting or just a setup issue?

For the setup:
I am installing the debian distro with proot-distro and login to it. I update the apt and install the snapd package.

**Comment by TomJo2000** [comment, marked as answer]
The error message already answers the question, but I'll add some additional context.

Snapd has hard dependencies on the Systemd init system.
Since Android doesn't use systemd as its init system there is no way to run Snapd on any Android device,
including when used as a host for Proot system.

Similar issues also apply to Flatpak, though in that case related to I believe Kernel C-Groups.
Don't quote me on that.

**Reply by matter1-git** [reply]
Thank you for your answer
